sr_utilities/scripts/sr_utilities/hand_finder.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+#
+# Copyright 2014 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation, either version 2 of the License, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import rospy
+import rospkg
+
+
+class HandControllerTuning(object):
+    def __init__(self, mapping):
+        """
+
+        """
+        ros_pack = rospkg.RosPack()
+        ethercat_path = ros_pack.get_path('sr_ethercat_hand_config')
+        self.friction_compensation = {}
+        self.host_control = {}
+        self.motor_control = {}
+        for hand in mapping:
+            self.friction_compensation[hand] = ethercat_path + '/controls/' \
+                + 'friction_compensation.yaml'
+            host_path = ethercat_path + '/controls/host/' + mapping[hand] + '/'
+            self.host_control[hand] = \
```

Comment:
All these dictionaries will have the serial number of the hand as the key.

Isn't this a bit strange from the user point of view? It seems better to use `mapping[hand]` (i.e. rh or lh) instead.